obal/data/roles/setup-sources/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+---
+- set_fact:
+    pkg_dir: "{{ inventory_dir }}/packages/{{ inventory_hostname }}"
+    annex_opts: annex_opts
+
+- name: get spec file
+  find:
+    pattern: "*.spec"
+    path: "{{ pkg_dir }}"
+  register: spec_file
+  tags:
+    - always
+
+- set_fact:
+    spec_file_path: "{{ spec_file.files[0].path }}"
+  tags:
+    - always
+
+- name: 'Extract Source0 from spec'
```

Comment:
What happens if I have more than one source that is annexed? Rare, but possible.

Reply:
In this case, it will be ignored.  Based on my reading of the setup_sources.sh file I was basing this off of, this was the only case.  I can go back and add in support for multiple sources

Reply:
https://github.com/theforeman/foreman-packaging/blob/rpm/develop/setup_sources.sh does loop over the files

Reply:
"our" version was ignoring other sources so far. how does the foreman code cope with non-remote files, or is `[ -h ... ]` enough to catch those?

Reply:
I haven't seen issues with the actually included tarballs we have with bundled installations. According to ```man test```:
> -h FILE
> FILE exists and is a symbolic link (same as -L)

That implies it's never an issue with actual files.